--- libksba-0.9.0/src/ksba.m4.wiget	2003-12-18 10:07:51.585464351 +0100
+++ libksba-0.9.0/src/ksba.m4	2003-12-18 10:08:04.584343259 +0100
@@ -14,7 +14,7 @@
 dnl              [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
 dnl Test for libksba and define KSBA_CFLAGS and KSBA_LIBS
 dnl
-AC_DEFUN(AM_PATH_KSBA,
+AC_DEFUN([AM_PATH_KSBA],
 [ AC_ARG_WITH(ksba-prefix,
             AC_HELP_STRING([--with-ksba-prefix=PFX],
                            [prefix where KSBA is installed (optional)]),
